File: Zadaca_5.2/Category.cs
using System;
using System.Collections.Generic;

namespace MVC
{
    public partial class Category
    {
        public byte CategoryId { get; set; }
        public string? CategoryName { get; set; }
        public byte? CategoryActive { get; set; }
        public byte? CategoryStatus { get; set; }
    }
}

File: Zadaca_6/types/Category.js
const {
  GraphQLObjectType,
  GraphQLInt,
  GraphQLString,
  GraphQLList,
} = require("graphql");
const { dbQuery } = require("../database");
const BrandType = require("./Brand");
const CategoryType = new GraphQLObjectType({
  name: "Category",
  fields: () => ({
    category_id: { type: GraphQLInt },
    category_name: { type: GraphQLString },
    category_active: { type: GraphQLInt },
    category_status: { type: GraphQLInt },
    brand: {
      type: BrandType,
      resolve: async (category) => {
        let res = await dbQuery(
          `SELECT * FROM Brands WHERE brand_id = ${parseInt(
            category.category_id
          )}`
        );
        return res[0];
      },
    },
  }),
});

module.exports = CategoryType;
